src/browser/browser.ts

```diff
@@ -122,7 +136,13 @@ export class Browser {
       page = await browser.newPage();
       this.addPageListeners(page);
 
-      return await this.takeScreenshot(page, options);
+      switch (options.renderType) {
+        case RenderType.CSV:
+          return await this.exportCSV(page, options);
```

Comment:
```suggestion
          return this.exportCSV(page, options);
```
`await` is redundant if you're just returning the promise.

Reply:
Actually, I just discovered that this is not redundant in a try/(catch)/finally clause. If we remove this, the finally code will be executed before the function ends and the page will be closed too soon.

Reply:
True, good point. Didn't notice this was used inside `try/catch`.